helpers/measures.ts

```diff
@@ -56,6 +56,23 @@ export function isDiffMetric(metricKey: string): boolean {
   return metricKey.indexOf('new_') === 0;
 }
 
+/*
+ * Conditional decimal count for QualityGate-impacting measures
+ * (e.g. Coverage %)
+ */
+export function getDecimalsNumber(threshold: number | undefined, value: number) {
```

Comment:
I am confused with the name of the function, and from the tests I don't understand well what it does (without reading the code).

Reply:
to be honest I just moved it out of sonarqube into sonar-ui-common
I'll rewrite the comment to make it clearer

Reply:
Ok ! :)